bedux.js

```diff
@@ -30,17 +30,25 @@ export function unsubscribeState(cb) {
     stateSubscribers = stateSubscribers.filter((registeredCb) => registeredCb != cb); 
 }
 
-export function unsubscribePartialState(cb, prop) {
+export function unsubscribePartialState(prop, cb) {
     if (typeof partialStateSubscribers[prop] !== 'undefined') {
         partialStateSubscribers[prop] = partialStateSubscribers[prop].filter((registeredCb) => registeredCb != cb); ;
     }
 }
 
-//Use in case of emergency, normally state shoud be accessed trough actions
+//Use in case of emergency, normally state should be accessed trough actions
 export function getState() {
     return {...state};
 }
 
+function getNestedValue(obj, key) {
+    return key
+        .split('.')
+        .reduce((nestedValue, partialKey) => {
+            return nestedValue.hasOwnProperty(partialKey) ? nestedValue[partialKey] : getNestedValue;
```

Comment:
I always try to avoid null/undefined as a return value, but I reckon that if the key does not exist it's better to acknowledge it and communicate it clearly instead of returning a fake and misleading value.